neo/Network/P2P/Peer.cs

```diff
@@ -37,11 +39,22 @@ private class WsConnected { public WebSocket Socket; public IPEndPoint Remote; p
         protected readonly ConcurrentDictionary<IActorRef, IPEndPoint> ConnectedPeers = new ConcurrentDictionary<IActorRef, IPEndPoint>();
         protected ImmutableHashSet<IPEndPoint> UnconnectedPeers = ImmutableHashSet<IPEndPoint>.Empty;
         protected ImmutableHashSet<IPEndPoint> ConnectingPeers = ImmutableHashSet<IPEndPoint>.Empty;
-
+        protected HashSet<IPAddress> TrustedIpAddresses { get; } = new HashSet<IPAddress>();
+        
         public int ListenerPort { get; private set; }
-        protected abstract int ConnectedMax { get; }
-        protected abstract int UnconnectedMax { get; }
-        protected virtual int ConnectingMax => ConnectedMax - ConnectedPeers.Count;
+        public int MinDesiredConnections { get; private set; } = DefaultMinDesiredConnections;
+        public int MaxConnections { get; private set; } = DefaultMaxConnections;
+        protected int UnconnectedMax { get; } = 1000;
+        protected virtual int ConnectingMax
```

Comment:
Why not just `MinDesiredConnections - ConnectedPeers.Count`?

Reply:
Firstly, if you are allowing unlimited max connections, do you really want to limit to only 10 at a time? Second, Why allow so few simultaneous connections on a potentially powerful machine? I think this is at least a little better than `MinDesiredConnections - ConnectedPeers.Count`